integration/class-jetpack.php

```diff
@@ -104,7 +104,7 @@ public static function add_reader_link( $actions, $item ) {
 		return array_merge(
 			array(
 				'reader' => sprintf(
-					'<a href="%s" target="_blank">%s</a>',
+					'<a href="%s" target="_blank" class="activitypub-external">%s</a>',
```

Comment:
You may be able to address the a11y issues by following the markup used in core:
https://github.com/WordPress/wordpress-develop/blob/6b6d2d1975cf72ace0f37ba440d380e16bdbb0d2/src/wp-admin/includes/dashboard.php#L1313

As an added bonus, if you use the `dashicons dashicons-external` classes you may not need to add any custom CSS, and you could leave Core handle everything.

Reply:
But if it is the WordPress way ;)

It is at least now consistent with the Site-Health.

<img width="817" height="327" alt="Screenshot 2025-09-30 at 13 30 44" src="https://github.com/user-attachments/assets/a3f8546f-6c8b-42f6-8f18-366e7269f50f" />